lib/screens/home/home_screen.dart

```diff
@@ -1,13 +1,134 @@
+    const ShopSlider(),
+    const ShopSlider(),
+    const ShopSlider(),
+    const ShopSlider(),
+  ];
+
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: Center(
-        child: Text("Home Screen"),
+    return Scaffold(
+      body: SafeArea(
+        child: Container(
+          margin: const EdgeInsets.symmetric(horizontal: 20.0, vertical: 12.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              Row(
+                children: [
+                  Text(
+                    'Shop',
+                    style: FontManager.raleway.copyWith(
+                      color: ColorManager.black2Color,
+                      fontSize: 28,
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                  const SizedBox(
+                    width: 20.0,
+                  ),
+                  Expanded(
+                    child: SizedBox(
+                      height: 40.0,
+                      child: TextField(
+                        onTap: () {},
+                        cursorHeight: 22.0,
+                        cursorColor: ColorManager.primaryColor,
+                        decoration: InputDecoration(
+                          border: const OutlineInputBorder(
+                            borderRadius: BorderRadius.all(
+                              Radius.circular(30.0),
+                            ),
+                            borderSide: BorderSide.none,
+                          ),
+                          filled: true,
+                          fillColor: ColorManager.primaryColor.withOpacity(0.1),
+                          hintText: 'Search',
+                          contentPadding:
+                              const EdgeInsets.only(top: 12, left: 16),
+                          hintStyle: FontManager.raleway.copyWith(
+                            fontSize: 16,
+                            color: ColorManager.primaryColor,
+                            fontWeight: FontWeight.w500,
+                          ),
+                          suffixIcon: Padding(
+                            padding: const EdgeInsets.symmetric(
+                              horizontal: 16.0,
+                            ),
+                            child: SvgPicture.asset(
+                              'assets/svgs/camera icon.svg',
+                              width: 16.0,
+                              height: 16.0,
+                            ),
+                          ),
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+              const SizedBox(
+                height: 20,
+              ),
+              CarouselSlider(
+                items: slidersList,
+                carouselController: _controller,
+                options: CarouselOptions(
```

Comment:
Use viewportFraction attribute to take the full width of the screen. default is 0.8